beakerx_tabledisplay/js/src/dataGrid/row/RowManager.ts

```diff
@@ -14,7 +14,7 @@
  *  limitations under the License.
  */
 
-import { filter, toArray } from '@lumino/algorithm';
```

Comment:
If I'm reading right, I believe [toArray](https://lumino.readthedocs.io/en/latest/api/functions/algorithm.toArray.html) is deprecated (also not doing much?) so that should be fine, but why was this breaking the table display?

Reply:
`toArray` given an iterator was not supported in Lumino v1, so it was breaking the JupyterLab v3 case.